src/App.js

```diff
@@ -1,13 +1,38 @@
-import React from "react";
+import React, { useState } from "react";
 
+import Restaurant from "./Restaurant";
 import Bookings from "./Bookings";
+import Heading from "./myComponents/Heading";
+import TouristInfoCards from "./myComponents/TouristInfoCards";
+import Footer from "./myComponents/Footer";
+import ThemeContext, { themes } from "./myComponents/ThemeContext";
+
 import "./App.css";
 
+const footer = [
+  "123 Fake Street, London, E1 4UD",
+  "hello@fakehotel.com",
+  "0123 456789"
+];
+
 const App = () => {
+  const [theme, setTheme] = useState(themes.light);
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
-      <Bookings />
+      {/* <header className="App-header">CYF Hotel</header> */}
+
+      <ThemeContext.Provider value={theme}>
```

Comment:
Do you need to use `ThemeContext.Provider` here? Is it over complicating here?

Reply:
No really, I just wanted to practice ThemeContext in the functional components and class components to change a theme to dark and light mode